WordPress/Classes/ViewRelated/Tools/Time Zone/Views/TimeZoneTableViewCell.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+class TimeZoneTableViewCell: WPTableViewCell {
+
+    let titleLabel = UILabel()
+
+    let leftSubtitle = UILabel()
+
+    let rightSubtitle = UILabel()
```

Comment:
would it work if we instantiate and setup these UI elements with a closure directly here? It could make the code more legible

Reply:
Sounds good - that's done as well!